### ISSUE TITLE
Launch error

### ISSUE THREAD

**Issue by overflow22** [ValveSoftware/Dota2-Gameplay#15304]
### Description

Everytime I launch dota this error is being shown and the game just closes.

### Screenshots

![error](https://github.com/ValveSoftware/Dota2-Gameplay/assets/151083281/967abe23-321a-47c0-86ad-ddc4716c7762)

**Comment by Krhandota2** [comment]
same problem u solve this bro??

**Comment by overflow22** [comment]
> same problem u solve this bro?
not yet, i tried reinstalling dota but the error wasn't fixed

**Comment by Krhandota2** [comment]
Yeah Many times i do it can u use 3060 nvida ?? Maybe re install driver

**Comment by overflow22** [comment]
i fixed the error. this is what i did: uninstalling dota2, deleting dota 2 beta folder and then installing dota on another hard disk (or create new partition of your hard disk and install it there).
